src/robusta/core/reporting/blocks.py

```diff
@@ -130,6 +130,35 @@ def truncate_content(self, max_file_size_bytes: int) -> bytes:
         return "\n".join(truncated_lines).encode("utf-8")
 
 
+class EmptyFileBlock(FileBlock):
```

Comment:
I think this should not extend a FileBlock
The point of not making it a FileBlock, was to not send it to all existing sinks.

But if this extends a FileBlock, it will be handled as a FileBlock by all sinks, right?

Reply:
done